dspace-api/src/main/java/org/dspace/content/dao/impl/BitstreamDAOImpl.java

```diff
@@ -77,8 +77,7 @@
     public Iterator<Bitstream> findByCommunity(Context context, Community community) throws SQLException {
         Query query = createQuery(context, "select b from Bitstream b " +
                 "join b.bundles bitBundles " +
-                "join bitBundles.bundle bundle " +
-                "join bundle.items item " +
+                "join bitBundles.items item " +
```

Comment:
Good remark, this was indeed an API bug fix. It will be removed from this PR and has already been placed in https://github.com/DSpace/DSpace/pull/1191 instead